content/en/about/why-falco.md

```diff
@@ -152,7 +152,7 @@ By using the same language to define all security policies and alerts, it become
 
 ##### Compatibility with Container Orchestration Tools
 
-We could write something related to easily exporting Falco metrics to Prometheus and Grafana. If there’s not a good example resource in place, we can just link prospective users to the kind page for setting-up a test cluster with [Falco, Prometheus and Grafana](https://falco.org/blog/falco-kind-prometheus-grafana/)
+At its core, Falco is a kernel event monitoring and detection agent that captures events, such as syscalls, based on custom rules. Falco can enhance these events by integrating metadata from the container runtime and Kubernetes. This tight integration with various container orchestration tools enables the expansion of Falco's detection capabilities and scope. It can also detect and alert on new containers and workloads being deployed, ensuring that security visibility is comprehensive across your entire infrastructure. In addition, through Falco's native support for daemonset-like deployments, it can seamlessly integrate into your existing setup, whether you are using Kubernetes, Docker Swarm, or other container orchestration platforms.
```

Comment:
> At its core, Falco is a kernel event monitoring and detection agent that captures events, such as syscalls, based on custom rules.

This sentence is not technically incorrect but can be misinterpreted. The word `captures` might give the impression that we are creating capturing files, or that we are outputting syscalls. The sentence also gives the impression that we are only capturing them based on custom rules. I would simplify the sentence to:

`At its core, Falco is a kernel event monitoring and detection agent.`

> across your entire infrastructure

This part can also be misinterpreted. I would say on the safe side and remove the word `entire`.

```suggestion
At its core, Falco is a kernel event monitoring and detection agent. Falco can enhance these events by integrating metadata from the container runtime and Kubernetes. This tight integration with various container orchestration tools enables the expansion of Falco's detection capabilities and scope. It can, for example, detect and alert on new containers and workloads being deployed, ensuring that security visibility is comprehensive across your infrastructure. In addition, through Falco's native support for daemonset-like deployments, it can seamlessly integrate into your existing setup, whether you are using Kubernetes, Docker Swarm, or other container orchestration platforms.
```

Reply:
I second @pmusa :+1:  

In particular, I agree to remove the "captures" and "rules" since they can be misinterpreted and it's not relevant here since this paragraph mainly focuses on the compatibility with container orchestrators

Reply:
This is great @pmusa would you like to update the main Falco readme as well since we have that sentence there currently? Just so we have everything cleaned up in one swing, I can quickly approve it.